readme.md

```diff
@@ -1,92 +1,40 @@
-
-    $ heroku create --stack cedar
-    $ heroku addons:create mongolab:sandbox
-    $ heroku addons:create heroku-redis:hobby-dev
-    $ git push heroku master
-    $ heroku open
-
-## Deploying on Debian-based OSes (Ubuntu etc)
-
-1) Install mongodb, redis servers and npm: 
-
-       $ sudo apt-get install mongodb redis npm
-
-2) If you don't want both the redis and mongo servers running all the time (For ex. if you are just testing), disable both startup scripts (If you wish to leave both running automatically, skip to step 4):
-
-       $ sudo systemctl disable mongodb
-       $ sudo systemctl disable redis-server`
-
-3) And stop both mongo and redis processes that were started automatically by apt-get: 
-
-       $ sudo systemctl stop mongodb
-       $ sudo systemctl stop redis-server
+[API docs are here](https://speckleworks.github.io/SpeckleSpecs/) - they are a good overview of what you can do.
 
-4) Clone SpeckleServer and run npm to install the needed nodejs packages: 
+## Installation
 
-       $ git clone https://github.com/speckleworks/SpeckleServer.git
-       $ cd SpeckleServer
-       $ npm install
+The Speckle Server is a nodejs (`v.8+`, `latest stable` preffered) app.
 
-5) Follow the instructions in `.env-base` file to configure your server.
-    
-6) Start mongo (create a folder somewhere to store the db): 
+- Dependencies: [mongodb](https://en.wikipedia.org/wiki/MongoDB) and [redis](https://en.wikipedia.org/wiki/Redis). 
+- Clone this repo `git clone https://github.com/speckleworks/SpeckleServer.git`
+- Configuration: create a `.env` using `.env-base` as a template, and fill in the required information.
+- Run `npm install` and thereafter `npm run`.
 
-       $ mongodb --dbpath /path/to/some/folder
-    
-7) Start redis in another terminal: 
 
-       $ redis-server
- 
-8) Check that both mongo and redis are running OK and that you can connect to them with these two clients:
+#### ⚠️ **Warning**: if these instructions seem confusing and you want to use speckle in production, please consult someone with more experience! Incorrect configuration can have **security and reliability implications**. 
 
-       $ mongo
-       $ redis-cli
-    
-9) Start Speckle in a third terminal: 
+#### 🍰 If you just want to try out speckle, at your own risk, you can use `https://hestia.speckle.works/api`.
 
-       $ node server.js
+**Note**: hestia is hosted on Digital Ocean. If you want to signup to digital ocean and support the running costs of speckle, use [the following affiliate link](https://m.do.co/c/947a2b5d7dc1) to get $100 in credit. 
 
-## API
-[API docs are here](https://speckleworks.github.io/SpeckleSpecs/) - they are a good overview of what you can do with the speckle server.
+To install a front-end plugin, such as the [admin ui](https://github.com/speckleworks/SpeckleAdmin), clone the respective repo in the `plugins` folder of the server. For example: 
 
-## Current Limitations
+- `cd server_install_folder/plugins`
```

Comment:
I think we might not need these instructions spelled out. The previous sentence explains the plugin process enough for the target audience.

Reply:
true enough, let's not get guilty again of the same issues we're trying to solve, but in a different place